Header header

sensor_msgs/CameraInfo cam_info
geometry_msgs/Pose cam_pose
sensor_msgs/RegionOfInterest[] roi